framework/contrib/ice_updater/include/LibcurlUtils.h

```diff
@@ -0,0 +1,134 @@
+/*******************************************************************************
```

Comment:
> I haven't looked but can we redistribute Curl sources ourselves?

I don't think we want to mess with this.  There are different ways of building it (OpenSSL vs. TLS) I just don't want to deal with.  It is released under the MIT/X license, which means it can be used in commercial products.